File: lib/gurobi605/linux64/examples/R/lp2.R
# Copyright 2014, Gurobi Optimization, Inc.
#
# Formulate a simple linear program, solve it, and then solve it
# again using the optimal basis.

library("gurobi")

model <- list()

model$A     <- matrix(c(1,3,4,8,2,3), nrow=2, byrow=T)
model$obj   <- c(1,2,3)
model$rhs   <- c(4,7)
model$sense <- c('>=', '>=')

# First solve - requires a few simplex iterations

result <- gurobi(model)

model$vbasis <- result$vbasis
model$cbasis <- result$cbasis

# Second solve - start from optimal basis, so no iterations

result <- gurobi(model)

File: lib/gurobi605/linux64/examples/matlab/lp2.m
% Copyright 2014, Gurobi Optimization, Inc.
%
% Formulate a simple linear program, solve it, and then solve it
% again using the optimal basis.

clear model;
model.A = sparse([1 3 4; 8 2 3]);
model.obj = [1 2 3];
model.rhs = [4 7];
model.sense = ['>' '>'];

% First solve requires a few simplex iterations
result = gurobi(model)

model.vbasis = result.vbasis;
model.cbasis = result.cbasis;

% Second solve - start from an optimal basis, so no iterations

result = gurobi(model)
